src/gobject/auto/enums.rs

```diff
@@ -0,0 +1,7 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
```

Comment:
Is it normal to have this empty file?

Reply:
Yes

Reply:
Can we not have it if it's useless? :)

Reply:
Sure, same for flags I guess :) But can we do that in a separate PR, it requires some further gir changes but I can put that on my list then

Reply:
Fine by me!